src/client/scripts/createRoom.js

```diff
@@ -1,6 +1,12 @@
 /* globals Rooms:false RoomMethods:false SnackbarMethods:false Room:false*/
 "use strict";
 
+// default categories
+var categories = [{category:"Went Well", color:"#00ff00"},
+                {category:"Went Poorly", color:"#ff0000"}];
+var categoriesDep = new Tracker.Dependency();
```

Comment:
What this is:
this updates the template whenever .changed() is called on it.  This allows categories to be neither a session (Session.set) nor be written to the DB until the room is created, and the template will still be up to date. pretty cool/useful imo